base/db/DESCRIPTION

```diff
@@ -73,4 +73,4 @@ Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.1
```

Comment:
Roll back roxygen version

Reply:
This applies to everywhere this change occurred. Roygen also needs to be manually forced to rerun the documentation with v 7.0.2 otherwise all the Rd files you built won't get reverted.